gateway-ha/src/main/java/io/trino/gateway/ha/clustermonitor/TrinoHealthStateType.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.gateway.ha.clustermonitor;
+
+public enum TrinoHealthStateType
+{
+    PENDING, // when clusters are switched from inactive to active
```

Comment:
question (non-blocking): to better understand for myself, is the only usage of `PENDING` right now to differentiate `PENDING` and `UNHEALTHY` on the UI/observability?

but functionally `PENDING` would act as `UNHEALTHY` in the Gateway? (as most tests are X = HEALTHY)

(this is fine I'm just reaffirming)

Reply:
correct

Reply:
can you write this to a comment code maybe at line 15?

Reply:
as in write the purpose of this enum class?

Reply:
just to clarify that PENDING is there for ui/observability purpose and functionally is unhealthy ? unless you plan to have other meaning for pending

Reply:
sure, i will add that